sweagent/agent/models.py

```diff
@@ -227,7 +227,9 @@ def __init__(self, args: ModelArguments, commands: list[Command]):
             self.api_model = cfg["AZURE_OPENAI_DEPLOYMENT"]
             self.client = AzureOpenAI(api_key=cfg["AZURE_OPENAI_API_KEY"], azure_endpoint=cfg["AZURE_OPENAI_ENDPOINT"], api_version=cfg.get("AZURE_OPENAI_API_VERSION", "2024-02-01"))
         else:
-            self.client = OpenAI(api_key=cfg["OPENAI_API_KEY"])
+            self.api_base_url = cfg.get("OPENAI_BASE_URL", None)
```

Comment:
seems like this should be a local variable, right? Or why do you set it as an attribute?

Reply:
well if you look at the open ai code you actually can set this as an environment variable, but it is unintuitive when using it in this project since the existing implementation of settings and variables are in the keys.cfg file

Reply:
No, I mean that it should be `api_base_url = ...` instead of `self.api_base_url`